docs/openapi3.yaml

```diff
@@ -2277,3 +2418,38 @@ components:
         percentage:
             type: number
             description: benchmark percentage weight
+    webhook:
+      type: object
+      required:
+        - name
+        - webhook_url
+        - events
+      properties:
+        id:
+          description: Unique webhook identifier
+          type: string
+          format: uuid
+          readOnly: true
+        name:
+          type: string
+          description: Webhook name
+        webhook_url:
+          type: string
+          description: Webhook url to post events
+        events:
+          description: list of events which will trigger the webhook
+          type: array
+          items:
+            $ref: '#/components/schemas/webhooks_types'
+
+    webhooks_types:
+      type: string
+      enum:
+        - started
```

Comment:
i would also add `results_interval` which we can configure to be sent at any time interval

Reply:
let's push this to phase2 and leave it out for now